src/url-loader/fs-url-loader.ts

```diff
@@ -30,15 +30,13 @@ export class FSUrlLoader implements UrlLoader {
   root: string;
 
   constructor(root: string = '') {
-    if (root.endsWith('/')) {
```

Comment:
Seemed like this was both wrong (shouldn't it be **not** endsWith?) and also unnecessary, because resolve normalizes the slash away anyway.